client/state/automated-transfer/selectors.js

```diff
@@ -68,3 +69,21 @@ export const isEligibleForAutomatedTransfer = compose(
 	getEligibilityStatus,
 	getEligibility
 );
+
+/**
+ * Determine if eligibility warnings or holds exist for
+ * a site other than a business plan hold.
+ *
+ * @param {Object} state global app state
+ * @param {number} siteId requested site for transfer info
+ * @returns {boolean} False if there is one and only one hold/warning referring to missing business plan
+ */
+export const hasEligibilityMessagesOtherThanBusiness = ( state, siteId ) => {
```

Comment:
Yes, this selector makes me uncomfortable, both in the way that it is named and its oddly specific nature. I initially approached this problem by using a couple of more general selectors.

Our actual goal, however, is to know when to show the eligibility card in the theme upload page. We only want to show the card when there are some warnings or holds to display. But in the case where the only message is the missing business plan, we show a nudge instead. So this selector contains logic that either needs to go here or in the component.